core/mo-services-xml/src/main/resources/xml/ServiceDefPLATFORM.xml

```diff
@@ -700,6 +700,29 @@
             <mal:type area="MAL" list="false" name="Float"/>
           </mal:field>
         </mal:composite>
+        <mal:composite comment="Constant velocity Ground Target Tracking attitude mode defined by the consumer." name="AttitudeModeTargetTrackingLinear" shortFormPart="12">
+          <mal:extends>
+            <mal:type area="Platform" list="false" name="AttitudeMode" service="AutonomousADCS"/>
+          </mal:extends>
+          <mal:field canBeNull="false" comment="The start latitude of the target to be tracked. [deg]" name="latitude_start">
```

Comment:
MAL composite structure fields names are called using lowerCamelCase convention

Reply:
Will fix that!
```suggestion
          <mal:field canBeNull="false" comment="The start latitude of the target to be tracked. [deg]" name="latitudeStart">
```